docs/triggers.md

```diff
@@ -51,6 +51,28 @@ spec:
     ref: pipeline-template
 ```
 
+Triggers can reference templates and bindings based on data in the event body, headers, or processed extensions. This allows contextual information to be pulled in from the event to determine which binding or template should be populated. The `ref` field for `bindings` and `templates` can be populated in the same manner as triggertemplate params.
+
+This can be useful when different pipelines require different resources or workspaces to run. `TriggerTemplate` objects by themselves must have a static configuration structurually, even if individual strings are able to be populated.
```

Comment:
```suggestion
This can be useful when different pipelines require different resources or workspaces to run. `TriggerTemplate` objects by themselves must have a static configuration structurally, even if individual strings are able to be populated.
```
Maybe just structure?

Reply:
updated to `structure` in the latest version